src/App.js

```diff
@@ -1,3 +1,67 @@
+import {useState} from 'react';
+import styled from 'styled-components';
+
+import InputDataDialog from './components/InputDataDialog.js';
+import { initialInputData } from "./db";
+
+
+
 export default function App() {
-  return <div>App goes here</div>;
+  
+  const [inputs, setInputs] = useState(initialInputData);
+  
+  let actualDate = new Date();
+  let formattedActualDate = actualDate.toLocaleDateString()
```

Comment:
If possible, you should use `const` instead of `let`. If I am seeing it correctly you do not assign another value to this variables later on, so you could use `const` here.

```suggestion
  const actualDate = new Date();
  const formattedActualDate = actualDate.toLocaleDateString()
```